internal/common/common.go

```diff
@@ -255,3 +255,36 @@ func GetHostNTPSources(db *gorm.DB, host *models.Host) (string, error) {
 	}
 	return infraEnv.AdditionalNtpSources, nil
 }
+
+func PlatformTypePtr(p models.PlatformType) *models.PlatformType {
+	return &p
+}
+
+func PlatformTypeValue(p *models.PlatformType) models.PlatformType {
+	if p == nil {
+		return ""
+	}
+	return *p
+}
+
+func ImageTypePtr(i models.ImageType) *models.ImageType {
+	return &i
+}
+
+func ImageTypeValue(i *models.ImageType) models.ImageType {
+	if i == nil {
+		return ""
+	}
+	return *i
+}
+
+func LogStatePtr(l models.LogsState) *models.LogsState {
+	return &l
+}
+
+func LogStateValue(l *models.LogsState) models.LogsState {
+	if l == nil {
+		return ""
+	}
+	return *l
+}
```

Comment:
I'd rather this not exist, but if it has to exist, can it go into its own file or package for type manipulations?

Reply:
OK